pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
Depends on the hardware. ARM NVIC works in level mode, and it won't care if the INT hardware clears early. After some slight debouncing,, the flag is set. Once it is set the hardware can't clear it... nor would you want to. If you have say, a rotary encoder, and it flies by too fast and say, misses a notch, that doesn't matter... an event **some event** triggered it. What the handler does to deal with the event is beyond the scope of the actual reason it triggered.